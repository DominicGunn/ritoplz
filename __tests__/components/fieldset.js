'use strict'

/* eslint-env jest */

import renderer from 'react-test-renderer'

import Fieldset from '../../components/ui/fieldset'

describe('ui fieldset component', () => {
  describe('snapshots', () => {
    it('should render fieldset', () => {
      const tree = renderer
        .create(
          <Fieldset title="title" description="description">Fieldset</Fieldset>
        )
        .toJSON()
      expect(tree).toMatchSnapshot()
    })
  })
})
