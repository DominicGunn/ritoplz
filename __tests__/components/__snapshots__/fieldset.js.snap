// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ui fieldset component snapshots should render fieldset 1`] = `
<div
  className="fieldset"
  data-jsx={1892067607}
>
  <div
    className="fielset-group--left"
    data-jsx={1892067607}
  >
    <h4
      data-jsx={1892067607}
    >
      title
       
    </h4>
    <p
      data-jsx={1892067607}
    >
      description
    </p>
  </div>
  <div
    className="fielset-group--right"
    data-jsx={1892067607}
  >
    Fieldset
  </div>
</div>
`;
